youtube/services.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from .models import Video
+from DjangoTut.settings import YOUTUBE_API_ACCESS_KEY
+
+
+class YoutubeService:
+    API_KEY = None
+    METHOD_GET = 'get'
+    METHOD_POST = 'post'
+    METHOD_DELETE = 'delete'
+
+    def __init__(self):
+        self.BASE_API_URL = "https://www.googleapis.com/youtube/v3/"
```

Comment:
to można wpisać od razu jaką parameter klasy, wyżej 2 linijki, po prostu `BASE_API_URL = "https://www.googleapis.com/youtube/v3/"`